### DISCUSSION TITLE
Mind maps getting deleted when I don't want them deleted

### DISCUSSION THREAD

**Discussion by halfbeing** [freeplane/freeplane#754] [Help]
I have been having a problem with mind maps getting deleted when I don't want them deleted.

First of all, recently I did a "save as", but instead of creating an additional copy of the mind map, Freeplane replaced the mind map with the mind map with the new name.

And secondly, I am getting mind maps randomly disappearing. This has happened twice, with 5 mind maps disappearing in total (3 the first time, and 2 the second time). These mind maps were in a folder synced with my iDrive cloud storage, and I had been working on them on my Mac. I often get messages in Freeplane telling me that my mind maps have been altered by another application, when they haven't, and asking me if I want to replace them with the altered version. From what I understand, this behaviour has something to do with whatever goes on in a synced folder. However, neither this behaviour, nor the more serious problem of files I have been working on completely disappearing, occur with any other application.

Is this problem particular to iDrive sync? Is it safe to store Freeplane maps in OneDrive or iCloud Drive?

**Comment by sifran-github** [comment]
not sure about the exact cause, but tip: search your deleted maps in the auto backup folder.
to see the location of the folder, go to tools->preferences->environment->see the settings under "Automatic save" section.

**Reply by halfbeing** [reply]
Thank you! That is a very useful tip.

**Comment by euu2021** [comment]
I use Dropbox and never had any problem.
